temporalimage/t4d.py

```diff
@@ -485,8 +486,17 @@ def _jsonread_frameTiming(jsonfilename):
         # TimeZero: Time zero to which all scan and/or blood measurements have been adjusted to.
         # InjectionStart = json_dict['TimeZero']
         # ScanStart = json_dict['TimeZero']
+    return frameStart, frameEnd, json_dict
 
-        return frameStart, frameEnd, json_dict
+####################Fix Serialization using JSONEncoder#########################################
+
+from json import JSONEncoder
+
+class NumpyArrayEncoder(JSONEncoder):
```

Comment:
I'm surprised that you need to do this. I wonder if something changed in the implementation of the `pint` package. I would prefer to not use this approach, and instead make sure that numpy arrays are converted to lists in `json_dict` instead